projects/1.19/assets/heat-and-climate/dcs_climate/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+  "item.dcs_climate.food.crop_cherry_wild":"野樱桃",
+  "item.dcs_climate.food.crop_cherry_common":"梅子",
+  "item.dcs_climate.food.crop_cherry_rare":"桃子",
+  "item.dcs_climate.food.crop_camellia_wild":"山茶籽",
+  "item.dcs_climate.food.crop_camellia_common":"木荷籽",
+  "item.dcs_climate.food.crop_camellia_rare":"茶叶",
+  "item.dcs_climate.food.crop_cinnamon_common":"肉桂",
+  "item.dcs_climate.food.crop_cinnamon_rare":"牛油果",
+  "item.dcs_climate.food.crop_citrus_wild":"柚子",
+  "item.dcs_climate.food.crop_citrus_common":"柑橘",
+  "item.dcs_climate.food.crop_citrus_rare":"柠檬",
+  "item.dcs_climate.food.crop_erica_wild":"石南花",
+  "item.dcs_climate.food.crop_erica_common":"杜鹃花",
+  "item.dcs_climate.food.crop_erica_rare":"蓝莓",
+  "item.dcs_climate.food.crop_morus_wild":"桑葚",
+  "item.dcs_climate.food.crop_morus_common":"构树果",
+  "item.dcs_climate.food.crop_morus_rare":"橡胶木浆果",
+  "item.dcs_climate.food.crop_olive_wild":"白蜡翅果",
+  "item.dcs_climate.food.crop_olive_common":"橄榄",
+  "item.dcs_climate.food.crop_olive_rare":"桂花",
+  "item.dcs_climate.food.crop_palm_wild":"椰子",
+  "item.dcs_climate.food.crop_palm_common":"椰枣",
+  "item.dcs_climate.food.crop_palm_rare":"油棕",
+
+  "item.dcs_climate.food.food_amaranth":"藜麦",
+  "item.dcs_climate.food.food_oat":"燕麦",
+  "item.dcs_climate.food.food_rye":"黑麦",
+  "item.dcs_climate.food.food_barley":"大麦",
+  "item.dcs_climate.food.food_buckwheat":"荞麦",
+  "item.dcs_climate.food.food_sorghum":"高粱",
+  "item.dcs_climate.food.food_zizania":"菰米",
+  "item.dcs_climate.food.food_makomotake":"茭白",
+  "item.dcs_climate.food.food_rice":"米",
+  "item.dcs_climate.food.food_aroma_rice":"香米",
+  "item.dcs_climate.food.food_masa":"玉米粉",
+  "item.dcs_climate.food.food_wheat":"面粉",
+  "item.dcs_climate.food.food_starch":"淀粉",
+  "item.dcs_climate.food.food_agar":"琼脂",
+
+
+  "item.dcs_climate.food.food_butter":"黄油",
+  "item.dcs_climate.food.food_margarine":"人造黄油",
+  "item.dcs_climate.food.food_cheese":"奶酪",
+  "item.dcs_climate.food.food_offal":"动物内脏",
+  "item.dcs_climate.food.food_gelatine":"吉利丁",
+  "item.dcs_climate.food.food_rennet":"凝乳酶",
+  "item.dcs_climate.food.food_raw_frog":"生蛙腿",
+  "item.dcs_climate.food.food_raw_prawn":"生虾",
+  "item.dcs_climate.food.food_raw_squid":"生鱿鱼",
+  "item.dcs_climate.food.food_raw_roe":"生鱼子",
+  "item.dcs_climate.food.dust_fish_powder":"鱼粉",
+
+  "item.dcs_climate.food.food_fruit_jam":"果酱",
+  "item.dcs_climate.food.food_marmalade":"橘子果酱",
+  "item.dcs_climate.food.food_custard":"卡仕达酱",
+  "item.dcs_climate.food.food_anko":"红豆泥",
+
+  "item.dcs_climate.food.food_sausage_raw":"生香肠",
+  "item.dcs_climate.food.food_sausage_smoked":"烟熏香肠",
+
+  "item.dcs_climate.food.food_soysauce":"酱油",
+  "item.dcs_climate.food.food_miso":"味噌",
+  "item.dcs_climate.food.food_mixed_spices":"混合香料",
+
+  "item.dcs_climate.food.pack_milk":"牛奶盒",
+  "item.dcs_climate.food.pack_soy_milk":"豆奶盒",
+  "item.dcs_climate.food.pack_coconut_milk":"椰奶盒",
+  "item.dcs_climate.food.pack_cream":"奶油盒",

```

```suggestion
  "item.dcs_climate.food.pack_cream":"稀奶油盒",
```
```suggestion
  "item.dcs_climate.food.pack_cream":"动物奶油盒",
```
（烘培用的）奶油一般分为两种：`稀奶油`（`鲜奶油`、`动物奶油`）和`淡奶油`（`植物奶油`），也有用`鲜奶油`统称两种奶油的。
`动物奶油`对应日文`生クリーム`，`植物奶油`对应日文`ホイップクリーム`。

---

顺带一提安利一下[这位](https://space.bilibili.com/1976715)的视频，里面把西点相关的知识讲的很详细（虽然也不是啦